### DISCUSSION TITLE
Tech Talks: What is new in Ant Media Server v2.10.0?

### DISCUSSION THREAD

**Discussion by zerostack0** [ant-media/Ant-Media-Server#6499] [Announcements]
![Designer (1)](https://github.com/user-attachments/assets/4663b99e-ac73-4948-84a1-b5d04001e292)

🚀 Exciting News for Streaming Enthusiasts! 🚀

Join us as we dive into the latest and greatest features of Ant Media Server v2.10.0! Whether you're a developer, a content creator, or a streaming service provider, you won't want to miss this!

Key Details:

    📅 Date: Jul 25th
    ⏰ Time: 3:00 PM GMT+3
    🌐 Location: https://meet.antmedia.io/Conference/techtalks

🔗 Save the Date: Don't miss out! [Add to your calendar](https://calendar.google.com/calendar/event?action=TEMPLATE&tmeid=NXFxY3JkNWdmZmFpajAzOXA0NXJ1cGpvOHYgY19kZjQ5YzExZWMyNDhhYWUxMjQ3ZTIzNTFlZjBkNzU1OGNlNDgxYzg4MDZjMWFmNjRmYjIzZTQyMzIzNjQ2ZTZkQGc&tmsrc=c_df49c11ec248aae1247e2351ef0d7558ce481c8806c1af64fb23e42323646e6d%40group.calendar.google.com)

Looking forward to seeing you there! 💻

**Comment by brixxnatt** [comment]
waiting for tomorrow , i have lot of questions , and need lot of fixes

**Reply by zerostack0** [reply]
Hi @brixxnatt 

I didn't see you in the meeting. I hope you are doing well.

**Comment by brixxnatt** [comment]
Hello My friend , I missed it , I hope you have  a tech Talk very soon.
thanks
